docs/src/main/sphinx/release/release-436.md

```diff
@@ -0,0 +1,52 @@
+# Release 436 (10 Jan 2024)
+
+## General
+
+* {{breaking}} Require JDK 21.0.1 to run Trino. ({issue}`20010`)
+* Improve query plans by not generating redundant predicates. ({issue}`16520`)
+* Fix query failure when invoking the {func}`json_table` function. ({issue}`20122`)
+* Fix query hanging when referencing a row field with a [SQL routine](/routines) ({issue}`19997`).
+* Fix potential incorrect results when using the {func}`ST_Centroid` and
+  {func}`ST_Buffer` functions for tiny geometries (less than 1e-3). ({issue}`20237`)
+
+## Delta Lake connector
+
+* Add the `parquet_ignore_statistics` catalog session property. ({issue}`20228`)
+* Improve performance of queries with selective joins on partition columns. ({issue}`20261`)
+* Reduce number of requests made to AWS Glue when listing tables, schemas, or
+  functions. ({issue}`20189`)
+* Fix incorrect results when querying Parquet files containing page indexes when
+  the query has filters on multiple columns. ({issue}`20267`)
+
+## Hive connector
+
+* Allow reading 3-element unions when the sub-union is composed of compatible
```

Comment:
Not sure what that means..

Reply:
I am kind of a loss for how to word this. If it can't be explained simply, maybe it's not worth a release note?

Reply:
Maybe simply "Fix failure when reading certain Avro data with Union data types".